app/question.js

```diff
@@ -95,29 +153,26 @@ define(['jquery', 'app/messages', 'app/config', 'app/string'], function ($, mess
       * that it was almost correct.
       */
       var input = document.getElementById("answer").value;
-      var answer = items[currentItemIndex].item_answer;
-
+      var answer = items[currentItemIndex].answer;
       var difference = levenstein(input,answer);
 
       if (difference == 0) {
-        messages.show( "Well done!", "success", config.constant("FEEDBACK_DELAY") );
-        nextQuestion();
+        handleScoreIncrease();
+        messages.show( "Well done!", "success", config.constant("FEEDBACK_DELAY_CORRECT") );
       } else if (isWithinMarginOfError(answer, difference)) {
-        messages.show( "Almost there! Your answer: " + input + " - Expected answer: " + answer + " (" + difference + " letter" + string.pluralIfAppropriate(difference) + " difference)", "warning", config.constant("FEEDBACK_DELAY") );
-        currentItemIndex = (currentItemIndex + 1) % items.length;
+        messages.show( "Almost there! Expected answer: <b>" + answer + "</b> (" + difference + " letter" + string.pluralIfAppropriate(difference) + " difference)", "warning", config.constant("FEEDBACK_DELAY_INCORRECT") );
       } else {
-        messages.show( "Wrong answer! Expected answer: " + answer , "danger", config.constant("FEEDBACK_DELAY") );
-        currentItemIndex = (currentItemIndex + 1) % items.length;
+        messages.show( "Wrong answer! Expected answer: <b>" + answer + "</b>", "danger", config.constant("FEEDBACK_DELAY_INCORRECT") );
```

Comment:
Since we will have internationalization module, these messages will be moved to language files